brave/brave-core
# Intermittent crash in Confirmations due to invalid wallet

Resolves https://github.com/brave/brave-browser/issues/8535

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Confirm confirmation tokens are refilled
- Confirm users are rewarded for viewing an ad
- Confirm confirmations are cashed-out
- Confirm "Estimated rewards pending" and "Ad notifications received this month" in ads rewards show the correct values (also after a restarting the browser)

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
